### DISCUSSION TITLE
v3的谷歌相册怎样添加？

### DISCUSSION THREAD

**Discussion by KimmyLueng** [AlistGo/alist#2272] [Q&A]
如题

**Comment by KimmyLueng** [comment]
现在新版能添加谷歌相册，能显示共享影集、全部媒体、全部影集这三个文件夹，但里面的图片还是不能显示

**Comment by kingli55** [comment]
用谷歌云盘的同样的方法，能添加谷歌相册，能显示共享影集、全部媒体、全部影集这三个文件夹，但里面的图片还是不能显示。
